docs/DFAnumerics.rst

```diff
@@ -544,6 +547,37 @@ mean mesh velocity and then the implicit term which leads to:
 With :math:`C_{vis} = \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k^{old}\| A_{Lat}\frac{dt}{m}`
 
 
+Ata Artificial viscosity
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+An upwinding method based on Lax-Friedrichs scheme
+""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Shallow Water Equations are well known for being hyperbolic transport equations.
+They have the particularity of carrying a discontinuity which makes them numerically unstable.
+Thus a decentering in time allows to better capture the discontinuity.
+This can be done in the manner of the Lax-Friedrich scheme.
+This decentering in time is formally the same as adding a viscous force.
+Transposed to the SPH method, this viscous force aplied on a given particle :math:`k`
+can be expressed as follows:
+
+.. math::
+  \mathbf{F_{{viscosity}_k}} = \sum_{l} \frac{m_l}{\rho_l} \Pi_{kl}
+
+with :math:`\Pi_{kl} = \lambda_{kl}(\mathbf{u}_j - \mathbf{u}_i) \cdot \mathbf{n}_{ij}`
+
+where :math:`\mathbf{u}_{ij} = \mathbf{u}_i - \mathbf{u}_j` is the relative velocity
+between particle i and j, and :math:`\lambda_{ij} = \frac{c_i+c_j}{2}` with
+:math:`c_i = \sqrt{gh_i}`, the wave speed. The :math:`\lambda_{ij}` is obtained as
+turning expressions related to time and spacial discretization parameters
+into an expression on maximal speed between both particles in the Lax Friedrich
+scheme.
+
+CFL condition
+"""""""""""""
+
```

Comment:
I am not sure where we add this section. Remove it for now, I will take care of adding this when we get something meaningfull